kafka-plugins-0.10/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <artifactId>kafka-plugins</artifactId>
     <groupId>io.cdap.plugin</groupId>
-    <version>2.2.0</version>
+    <version>2.2.1</version>
```

Comment:
better for this to be snapshot, and update it when we bump the dependencies to non-snapshot